to_production.txt

```diff
@@ -11,6 +11,7 @@ ofl/buenard # https://github.com/google/fonts/pull/7573
 ofl/cormorantgaramond # https://github.com/google/fonts/pull/8593
 ofl/cormorantinfant # https://github.com/google/fonts/pull/8594
 ofl/englebert # https://github.com/google/fonts/pull/8688
+ofl/iansui # https://github.com/google/fonts/pull/9022
```

Comment:
New project that requires file size review before we push to Workspace editors

Reply:
Aaron confirmed that this is a single instance static format family that has a size of 9.2MB. This should be ok for Workspace.